applications/ShallowWaterApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -69,6 +69,12 @@ namespace Python
         .def("SetMeshPosition", &ShallowWaterVariablesUtility::SetMeshPosition)
         .def("SetElementsActive", &ShallowWaterVariablesUtility::SetElementsActive)
         ;
+
+    py::class_< EstimateDtShallow > (m, "EstimateDtShallow")
+        .def(py::init<ModelPart&, Parameters&>())
```

Comment:
```suggestion
        .def(py::init<ModelPart&, Parameters>())
```

parameters should be passed by-value according to Riccardo

Reply:
done